static/css/frame.scss

```diff
@@ -150,15 +150,16 @@
 }
 
 .promo-panel {
+  background: $white;
```

Comment:
have to add explicit bg declarations in a few spots since not everything gets a color-scheme helper class. Other changes here are just putting things in alphabetical order